python/cudf/cudf/core/column/column.py

```diff
@@ -412,7 +412,7 @@ def from_arrow(cls, array):
         ]
 
     def _get_mask_as_column(self):
-        return libcudf.transform.mask_to_bools(self)
+        return libcudf.transform.mask_to_bools(self.mask, 0, len(self))
```

Comment:
Any reason we shouldn't use `self.base_mask` and pass `self.offset` instead of 0 here?

Reply:
The `begin_bit` here is not an offset for the column. It is basically an index to start scanning the column mask from, for example:

```python
>>> import cudf
>>> s = cudf.Series([1, 2, 3, None, 5])
>>> s
0       1
1       2
2       3
3    <NA>
4       5
dtype: int64
<cudf.core.column.numerical.NumericalColumn object at 0x7fad2f16fa70>
>>> cudf.Series(cudf._lib.transform.mask_to_bools(s._column.mask, 0, 5))
0     True
1     True
2     True
3    False
4     True
dtype: bool
>>> cudf.Series(cudf._lib.transform.mask_to_bools(s._column.mask, 1, 5))
0     True
1     True
2    False
3     True
dtype: bool
```

Reply:
> Any reason we shouldn't use `self.base_mask`

But I think we wanted to have it `mask` according to this no?: https://github.com/rapidsai/cudf/pull/6291#discussion_r493857019

Reply:
Yea, so if we passed it as `libcudf.transform.mask_to_bools(self.base_mask, self.offset, self.offset + len(self))` it accomplishes the same thing as what you have without having to possibly copy construct a new bitmask if there's a column offset.

Reply:
> But I think we wanted to have it `mask` according to this no?: [#6291 (comment)](https://github.com/rapidsai/cudf/pull/6291#discussion_r493857019)

This is a libcudf API that specifically can handle an offset / size parameter as opposed to cupy which can't.